src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -53,6 +53,18 @@ module Make (Inputs : Inputs.S) :
    and type network := Inputs.Network.t = struct
   open Inputs
 
+  module Catchup_jobs = struct
```

Comment:
I don't think that could happen, but if it did, the ledger catchup would already throw an error right now since the ledger it's syncing to would either be a closed db or a mask on a closed db.